### ISSUE TITLE
Failing MarkdownRenderer tests

### ISSUE THREAD

**Issue by colinrubbert** [radar/twist#15]
I'm sure I'm just missing something here but I've tried multiple times to remove and reinstall/setup the repo on my local environment (Kubuntu Linux) and these three tests keep failing. 

Quick info: 
- redis is installed and running
- phantomjs is installed and working
- sidekiq is installed and running
- pulled latest branch, ran submodules, and setup the db

Here are the failures:

```ruby
cr@k-dev:~/code/rails_projects/twist$ bundle exec rspec spec
...........................FF.F.................

Failures:

  1) MarkdownRenderer can parse a titleized code listing
     Failure/Error: expect(parsed_code.css("div.highlight")).not_to be_empty
       expected `[].empty?` to return false, got true
     # ./spec/lib/markdown_renderer_spec.rb:150:in `block (2 levels) in <top (required)>'
     # ./spec/rails_helper.rb:40:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:39:in `block (2 levels) in <top (required)>'

  2) MarkdownRenderer maintains line breaks in code listing
     Failure/Error: expect(parsed_code.css("div.highlight")).not_to be_empty
       expected `[].empty?` to return false, got true
     # ./spec/lib/markdown_renderer_spec.rb:168:in `block (2 levels) in <top (required)>'
     # ./spec/rails_helper.rb:40:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:39:in `block (2 levels) in <top (required)>'

  3) MarkdownRenderer can parse a titleized code listing with a paragraph following
     Failure/Error: expect(parsed_code.css("div.highlight")).not_to be_empty
       expected `[].empty?` to return false, got true
     # ./spec/lib/markdown_renderer_spec.rb:212:in `block (2 levels) in <top (required)>'
     # ./spec/rails_helper.rb:40:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:39:in `block (2 levels) in <top (required)>'

Finished in 16.12 seconds (files took 2.58 seconds to load)
48 examples, 3 failures

Failed examples:

rspec ./spec/lib/markdown_renderer_spec.rb:131 # MarkdownRenderer can parse a titleized code listing
rspec ./spec/lib/markdown_renderer_spec.rb:154 # MarkdownRenderer maintains line breaks in code listing
rspec ./spec/lib/markdown_renderer_spec.rb:195 # MarkdownRenderer can parse a titleized code listing with a paragraph following
```
If anyone has some insight into this it would be amazing. Since I'm just starting I don't know how everything is setup and configured so I don't want to go around toiling with the code.

**Comment by radar** [comment]
Hi @colinrubbert, thanks for pointing this out. This was a mistake on my behalf, left over from when I was doing some CI server research. I've now fixed this test in 2c0e86e. If you pull from the latest master, you'll receive this commit.

**Comment by colinrubbert** [comment]
Excellent, thank you for getting it fixed so soon. Been slowly working through the book. Great timing, thanks!
